### PR TITLE
[Community Tutorial] Combining FeatureCollections

### PR DESCRIPTION
**Pull request by sabrinaszeto**

### CODE REVIEW

File: tutorials/combine-feature-collections/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+author: sabrinaszeto
+tags: beginner, featurecollection
+date_published: 2019-09-18
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/b4a2877c24cdd175bca2d7252cd733de)
+
+This basic tutorial shows how users can combine two `ee.FeatureCollection`s into a new `ee.FeatureCollection`.
+
+## Create two `ee.FeatureCollection` objects
```
Comment (gino-m) [inline tutorials/combine-feature-collections/index.md]: > Should all mentions of ee.Objects be in single backticks?

Ideally yes, for consistency with other docs in https://developers.google.com/earth-engine/tutorials.

File: tutorials/combine-feature-collections/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: Combine FeatureCollections into a new FeatureCollection
+description: This basic tutorial shows how users can combine two FeatureCollections into one. It is targeted at beginners.
+author: sabrinaszeto
+tags: beginner, featurecollection
+date_published: 2019-09-18
```
Comment (gino-m) [inline tutorials/combine-feature-collections/index.md]: Why is each line prefixed by "+"? I saw this in @TC25's PR as well. Ws this by accident or intentional?

**Comment by sabrinaszeto** [comment]
@googlebot I fixed it.

**Comment by sabrinaszeto** [comment]
@googlebot I fixed it.

**Comment by gino-m** [comment]
@sabrinaszeto The issue seems to be that the PR has commits not associated with your GitHub account. One way to fix this without recreating the branch would be to rebase right before the broken commit:
```
$ git rebase -i 25fbeb23a907d4c8b9421889d622037b98022749
```

Then, when the editor opens, replace "pick" with "edit before the three broken commits and save.

Then you'd do this three times:
```
$ git commit --amend --author="Sabriza Szeto <youremailaddress>"
$ git rebase --continue
``

Once the rebase is complete, you'll need to force push to your branch:
```
$ git push --force
```

If you get in trouble at any point you do this to abort:
```
$ git rebase --abort
```

Please let me know if this helps. If not, we can recreate the repo in a separate branch.

File: tutorials/combine-feature-collections/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    [-109.05, 41],
+    [-111.05, 41],
+    [-111.05, 42],
+    [-114.05, 42]
+]);
+```
+
+Then, generate two sets of different random points containing 25 points each. We ensure that the points are different by using a different seed, namely 12 and 1, to generate each set.
+```
+var NewFEATURES = ee.FeatureCollection.randomPoints(UtahGEOMETRY, 25, 12);
```
Comment (gino-m) [inline tutorials/combine-feature-collections/index.md]: Please use lowerCamelcase for all variable names (as opposed to MixedCASE or ALLCAPS)

File: tutorials/combine-feature-collections/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+[Open In Code Editor](https://code.earthengine.google.com/b4a2877c24cdd175bca2d7252cd733de)
+
+This basic tutorial shows how users can combine two `ee.FeatureCollection`s into a new `ee.FeatureCollection`.
+
+## Create two `ee.FeatureCollection` objects
+
+Let's begin by generating two sets of random points within the boundary of Utah state in USA. First, define the boundary of Utah as a geometry.
+
+```
+var UtahGEOMETRY = ee.Geometry.Polygon([
```
Comment (gino-m) [inline tutorials/combine-feature-collections/index.md]: Could you also create a file alongside the md with the entire runnable JavaScript source? (e.g., example.js)

File: tutorials/combine-feature-collections/index.md (hunk 3 above)
Comment (sabrinaszeto) [reply tutorials/combine-feature-collections/index.md]: Will work on this!

File: tutorials/combine-feature-collections/index.md (hunk 4 above)
Comment (sabrinaszeto) [reply tutorials/combine-feature-collections/index.md]: Will work on this as well.

**Comment by gino-m** [comment]
Hi @sabrinaszeto, do you have a rough ETA of when this can be finalized? I'd like to publish these in the next week or two if possible. Thanks for your great content!

**Comment by gino-m** [comment]
Superseded by #53.

**Pull request closed without merging**